### PR TITLE
kube-aws: improve success messages

### LINKED ISSUE #353: kube-aws: output success messages
Add two success messages for:

1. `kube-aws init`
2. `kube-aws render`

Existing init output:

```
$ kube-aws init --cluster-name=xxxxxx \
 --external-dns-name=xxxxx \
 --region=xxxx \
 --availability-zone=xxxx \
 --key-name=xxxxx \
 --kms-key-arn="arn:aws:kms:us-east-1:xxxxx"
Edit cluster.yaml to parameterize the cluster. Then use the "kube-aws render" command to render the stack template
````

What if we reformatted the bottom to say:

```
$ kube-aws init --cluster-name=xxxxxx \
 --external-dns-name=xxxxx \
 --region=xxxx \
 --availability-zone=xxxx \
 --key-name=xxxxx \
 --kms-key-arn="arn:aws:kms:us-east-1:xxxxx"
Success! Created cluster.yaml

Next steps:
1. Edit cluster.yaml to parameterize the cluster.
2. Use the "kube-aws render" command to render the stack template
3. Start the cluster with "kube-aws up"
```

Existing render output:

```
$ kube-aws render
Edit cluster.yaml and/or any of the cluster assets. Then use the "kube-aws up" command to create the stack
```

I'm not sure why we're suggesting to tweak the yml again. What about this:

```
$ kube-aws render
Success! Stack rendered to stack-template.json

Next steps:
1. Validate your changes to cluster.yaml with "kube-aws render"
2. Start the cluster with "kube-aws up"
```

### PR DESCRIPTION
**Pull request by cgag**
Before it wasn't entirely clear that the command had succeeded.  This
also clarifies if next steps are required or not, as well as makes them
easier to read.

Credit to robszumski for writing most of the messages.

fixes #353

### CODE REVIEW

File: multi-node/aws/cmd/kube-aws/command_init.go [hunk 1]
```diff
@@ -70,6 +70,13 @@ func runCmdInit(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Error exec-ing default config template: %v", err)
 	}
 
-	fmt.Printf("Edit %s to parameterize the cluster. Then use the \"kube-aws render\" command to render the stack template.\n", configPath)
+	successMsg :=
+		"Success! Created %s\n" +
```
Comment (colhom) [inline multi-node/aws/cmd/kube-aws/command_init.go]: Convention on this project is to use backticks for multi-line strings.

File: multi-node/aws/cmd/kube-aws/command_up.go [hunk 2]
```diff
@@ -72,7 +74,12 @@ func runCmdUp(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Failed fetching cluster info: %v", err)
 	}
 
-	fmt.Print(info.String())
+	successMsg := "Success! Your AWS resources have been created: \n" +
+		info.String() + "\n" +
```
Comment (colhom) [inline multi-node/aws/cmd/kube-aws/command_up.go]: Good idea here. There have been multiple issues filed asking about this

**Comment by colhom** [comment]
Change the multi-line strings to backticks, otherwise looks good.

**Comment by robszumski** [comment]
Example of the full output?

**Comment by cgag** [comment]
Full output:

2016-04-05-152946_946x525_scrot

Switched to backticks.

**Comment by robszumski** [comment]
LGTM. Nice work, it's much better!

**Comment by colhom** [comment]
This looks great. Thanks cgag

**Pull request merged**